igneous/tasks/skeletonization/skeletonization.py

```diff
@@ -88,9 +104,12 @@ def TEASAR(
       anisotropy=anisotropy, invalid_vertices=invalid_vertices,
     )
     valid_labels -= invalidated
-    paths.append(path)
     for vertex in path:
       invalid_vertices[tuple(vertex)] = True
+    path = np.concatenate((path[0:-2:path_downsample, :],
```

Comment:
Not sure the path downsampling is appropriate at this stage since it's configured to basically not affect anything other than path union. I kind of feel like that might be a CloudVolume feature, but as a convenience I think it's fine for now.